docs/editor/workspaces.md

```diff
@@ -9,39 +9,91 @@ MetaDescription: Learn about Visual Studio Code workspaces
 
-You don't have to do anything for a folder to become a VS Code "workspace" other than open the folder with VS Code. Once a folder has been opened, VS Code will automatically keep track of things such as your open files and editor layout so the editor will be as you left it when you reopen that folder. You can also manually add other folder specific configurations such as project-specific settings (versus global user settings) and task definition and debugging launch files.
+The concept of a workspace enables VS Code to scope useful application state such files that were opened or the editor layout and persist configurations such as workspace-specific [settings](/docs/getstarted/settings.md) and [tasks](/docs/editor/tasks.md).
 
-You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. If you think of a "workspace" as the root folder of a project that has extra VS Code knowledge, you will be correct in most cases. Depending on your development workflow, you can also have more than one folder in a "workspace", using an advanced configuration called [Multi-root workspaces](#multi-root-workspaces).
+You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. Think of a workspace as the root of a project that comes with extra VS Code knowledge and capabilities.
 
-## Workspace state
+## How do I open a VS Code "workspace"
+
+You don't have to do anything for a folder to become a VS Code workspace other than open the folder with VS Code. Once a folder or has been opened, VS Code will automatically keep track of things such as your open files and editor layout so the editor will be as you left it when you reopen that folder. You can also manually add other folder specific configurations such as workspace-specific settings (versus global user settings) and task definition and debugging launch files.
+
+![Single-folder workspace](images/workspaces/single-folder-workspace.png)
+
+*A single folder workspace opened inside VS Code*
+
+The same is true for multi-root workspaces that you can open easily by selecting the `<name>.code-workspace` file for opening.
+
+![Multi-folder workspace](images/workspaces/multi-folder-workspace.png)
 
+*A multi-root workspace opened in VS Code*
+
+**Note:** The visual difference of having a folder opened vs. opening a `.code-workspace` file can be very subtle. To give you a hint that a `.code-workspace` file has been opened, some pieces of the user interface (e.g. the root of the explorer) show an extra "(Workspace)" suffix next to the name.
+
+## Workspace state
```

Comment:
Agreed that we probably don't need this section or a detailed enumeration of persisted state.